cmd/clusterlink/cmd/deploy/deploy_peer.go

```diff
@@ -103,6 +116,14 @@ func (o *PeerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.Uint16Var(&o.IngressPort, "ingress-port", apis.DefaultExternalPort,
 		"Represents the ingress port. By default it is set to 443 for LoadBalancer"+
 			" and a random port in range (30000 to 32767) for NodePort.\nThis option is only valid if --autostart is set.")
+	fs.StringVar(&o.DataplaneType, "dataplane", platform.DataplaneTypeEnvoy,
+		"Type of dataplane, Supported values: \"envoy\" (default), \"go\"")
```

Comment:
might be able to use tick for string delimiter and avoid escaping quotes

Reply:
I prefer to use it like that because cobra print also the default value with "